jetty-core/jetty-xml/src/main/java/org/eclipse/jetty/xml/XmlAppendable.java

```diff
@@ -30,31 +33,31 @@
     private final Stack<String> _tags = new Stack<>();
     private String _space = "";
 
-    public XmlAppendable(OutputStream out, String encoding) throws IOException
+    public XmlAppendable(OutputStream out, Charset charset) throws IOException
     {
-        this(new OutputStreamWriter(out, encoding), encoding);
+        this(new OutputStreamWriter(out, charset), charset);
     }
 
     public XmlAppendable(Appendable out) throws IOException
     {
         this(out, 2);
     }
 
-    public XmlAppendable(Appendable out, String encoding) throws IOException
+    public XmlAppendable(Appendable out, Charset charset) throws IOException
     {
-        this(out, 2, encoding);
+        this(out, 2, charset);
     }
 
     public XmlAppendable(Appendable out, int indent) throws IOException
     {
-        this(out, indent, "utf-8");
+        this(out, indent, StandardCharsets.UTF_8);
     }
 
-    public XmlAppendable(Appendable out, int indent, String encoding) throws IOException
+    public XmlAppendable(Appendable out, int indent, Charset charset) throws IOException
```

Comment:
This constructor is not used outside of this class, can probably be removed/collapsed into the constructors that are used.